packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -304,6 +304,16 @@ mixin WidgetsBinding on BindingBase, SchedulerBinding, GestureBinding, RendererB
       );
     }
 
+    // Expose the ability to send Widget rebuilds as [Timeline] events.
+    registerBoolServiceExtension(
```

Comment:
This needs to be behind the kReleaseMode flag

Reply:
Done.

Reply:
It looks like this is outside of the flag still

Reply:
Oops, sending another patch.